aw_server/log.py

```diff
@@ -20,7 +20,7 @@ def log(self, levelname, message, *args):
         msg = args[0]
         code = args[1]
 
-        if code in [200, 304]:
+        if code in [200, 301, 304, 404]:
```

Comment:
What? It'd be interesting if a resource on the server was not found. If we get 404s somewhere we should eliminate them.